dns_server.go

```diff
@@ -113,6 +130,8 @@ func handleDnsRequest(l *logrus.Logger, w dns.ResponseWriter, r *dns.Msg) {
 
 func dnsMain(l *logrus.Logger, hostMap *HostMap, c *config.C) func() {
 	dnsR = newDnsRecords(hostMap)
+	dnsR.dnsWildcardEnabled = c.GetBool("dns.wildcard", false)
+	dnsR.dnsWildcardLimit = c.GetInt("dns.wildcard_limit", 5)
```

Comment:
@wadey I've just realised these should be `lighthouse.dns.wildcard` right?

Reply:
Yes that is correct https://github.com/slackhq/nebula/blob/v1.5.2/examples/config.yml#L32